Metadata/AmazonMetadataBuilder.php

```diff
@@ -13,7 +13,7 @@
 
 use Aws\S3\Enum\CannedAcl;
 use Aws\S3\Enum\Storage;
-use Guzzle\Http\Mimetypes;
+use Guzzle\Http\Mimetypes; # TODO: Find something common
```

Comment:
I don't think so, but I hardly think this should be a responsibility of a HTTP something.